index.html

```diff
@@ -75,8 +76,8 @@ <h2 class="stories"> Multi-Post Stories</h2>
       </article>
      </section>
 
-     <section id="boxes">
-      <div class="box2"> 
+     <section>
+  <div class="box2"> 
           <div class="box"> 
             <div class="tags1">
               <button class="projects1">See project</button>
```

Comment:
- [ ] Good job so far. However, there are few things you need to work on before you move on. The information about the first project under my work is missing on your design. Kindly look into it and fix. Please, take a look

### Your Design
![your-first-work](https://user-images.githubusercontent.com/26597432/206869724-5643e566-95e8-4f56-b664-b69f3da3f2d3.png)

### Figma Design
![figma-first-work](https://user-images.githubusercontent.com/26597432/206869734-992723cb-9449-445f-b6cb-ae69684ef0a6.png)